src/tests/test_alldebrid_downloader.py

```diff
@@ -46,91 +51,51 @@ def get(url, **params):
     yield downloader
 
     # tear down mock
-    alldebrid.get = get
-
-
-## Downloader tests
-def test_process_hashes(downloader):
-    hashes = downloader.process_hashes(["abc"], None, [False, True])
-    assert len(hashes) == 1
-
-
-def test_download_cached(downloader):
-    torrent_id = downloader.download_cached({"infohash": "abc"})
-    assert torrent_id == MAGNET_ID
-
+    alldebrid.AllDebridRequestHandler.execute = _execute
 
-def test_get_torrent_names(downloader):
-    names = downloader.get_torrent_names(123)
-    assert names == ("Ubuntu 24.04", None)
 
+## DownloaderBase tests
+def test_validate(downloader):
+    assert downloader.validate() == True
 
-## API parsing tests
-def test_get_instant_availability(instant):
-    alldebrid.get = instant
-    infohashes = [UBUNTU]
-    availability = get_instant_availability(infohashes)
-    assert len(availability[0].get("files", [])) == 2
 
+def test_get_instant_availability(downloader):
```

Comment:
This test is currently failing due to bugs in the downloader, @dreulavelle has a fix in his branch